python/_dynet.pyx

```diff
@@ -598,11 +604,12 @@ cdef class Expression: #{{{
     def __str__(self):
         return "exprssion %s/%s" % (<int>self.vindex, self.cg_version)
 
-    def __getitem__(self, int i):
-        return pick(self, i)
-
-    def __getslice__(self, int i, int j):
-        return pickrange(self, i, j)
+    # __getitem__ and __getslice__ in one for python 3 compatibility
```

Comment:
ok, thanks. will merge -- but now there are some merge conflicts :/ sorry about that.